Update `CWC 2022` Qualifiers rules, add links (#7346)
ppy/osu-wiki: wiki/Tournaments/CWC/2022/en.md

| :-- | :-- |
| Manager | ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578), ![][flag_US] [ChillierPear](https://osu.ppy.sh/users/9501251), ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779) |
| Mappool selector | ![][flag_HK] [autofanboy](https://osu.ppy.sh/users/636114), ![][flag_JP] [ekr](https://osu.ppy.sh/users/4497706), ![][flag_CN] [F D Flourite](https://osu.ppy.sh/users/2459589), ![][flag_FR] [Rougoku](https://osu.ppy.sh/users/6900166), ![][flag_CA] [wwwww](https://osu.ppy.sh/users/8434466), ![][flag_TN] [-Ken](https://osu.ppy.sh/users/4430811) |

> | Commentator | ![][flag_US] [Dohland](https://osu.ppy.sh/users/5220511), ![][flag_US] [Elux](https://osu.ppy.sh/users/12004983), ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508) ![][flag_AU] [KWYJIBO](https://osu.ppy.sh/users/7178386), ![][flag_US] [Lumenite-](https://osu.ppy.sh/users/6256027), ![][flag_AU] [Maitoo](https://osu.ppy.sh/users/16899553), ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164), ![][flag_NL] [Sartan](https://osu.ppy.sh/users/4100941), ![][flag_US] [Secre](https://osu.ppy.sh/users/2306637), ![][flag_US] [Snowleopard](https://osu.ppy.sh/users/3790227), ![][flag_US] [Zak](https://osu.ppy.sh/users/1375955), ![][flag_BE] [\[ Quinten \]](https://osu.ppy.sh/users/6283136) |

Reviewer: ```suggestion
| Commentator | ![][flag_US] [Dohland](https://osu.ppy.sh/users/5220511), ![][flag_US] [Elux](https://osu.ppy.sh/users/12004983), ![][flag_GB] [JBHyperion](https://osu.ppy.sh/users/4879508), ![][flag_AU] [KWYJIBO](https://osu.ppy.sh/users/7178386), ![][flag_US] [Lumenite-](https://osu.ppy.sh/users/6256027), ![][flag_AU] [Maitoo](https://osu.ppy.sh/users/16899553), ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164), ![][flag_NL] [Sartan](https://osu.ppy.sh/users/4100941), ![][flag_US] [Secre](https://osu.ppy.sh/users/2306637), ![][flag_US] [Snowleopard](https://osu.ppy.sh/users/3790227), ![][flag_US] [Zak](https://osu.ppy.sh/users/1375955), ![][flag_BE] [\[ Quinten \]](https://osu.ppy.sh/users/6283136) |
```

5. The mappool **must** be played in the following order: NM1, NM2, NM3, HD1, HD2, HR1, HR2, DT1, DT2.
6. Each team must have 3 players for each map. They can be exchanged freely after each map is concluded.
7. A team's final score for a map is considered as the highest combined team score for that map (i.e. the sum of each individual player's score). Only the highest score out of the two runs will be used for seeding.

> 8. The seeding method used for Qualifiers will be %MAX - the highest combined team score for each map will receive 100% of the points, and every other team will be awarded a percentage of that top score.

Reviewer: ```suggestion
8. The seeding method used for Qualifiers will be %MAX—the highest combined team score for each map will receive 100% of the points, and every other team will be awarded a percentage of that top score.
```

>    - The exact formula that will be used for all teams (and for each map) is `Final Score = {Team Score} / MAX({Map Score})`, where:
>      - `Final Score` is the percentage awarded to the current team
>      - `{Team Score}` is the score the current team achieved on the current map
>      - `MAX({Map Score})` is the highest score achieved for the current map.

Reviewer: ```suggestion
   - The exact formula that will be used for all teams (and for each map) is `Final Score = Team Score / MAX(Map Score)`, where:
     - `Final Score` is the percentage awarded to the current team
     - `{Team Score}` is the score the current team achieved on the current map
     - `MAX(Map Score)` is the highest score achieved for the current map
```

> 9. An optional 3-minute break will be offered between the first and second play-through of the mappool.

Reviewer: ```suggestion
9. An optional 3-minute break will be offered between the first and second playthrough of the mappool.
```

10. All teams will play their qualifiers in separate rooms. It is suggested that teams do not broadcast or share their results publicly to avoid seed manipulation.
11. Only the top 32 seeded teams will advance to the Round of 32.

> 12. The Qualifier stage results will be published in this wiki article after all teams have played.

Reviewer: i do think "rules" like this are redundant if it would make sense to remove them later anyway

```suggestion
```

Reviewer: and if it's more intended than not, then it'd at least make "match instructions 6." redundant/contradictory... ?

> 6. Results for the Qualifier stage will be published via a statistics sheet.

Author: Indeed, I will just merge both.
